### PR TITLE
crate index.js file and add a console log to it

### PR DESCRIPTION
**Pull request by rohan-sorkar**
created a new file inside the root directory called index.js
and add  a console log by using console.log();

### CODE REVIEW

**Review by mohammadsaju — COMMENTED** [review]
can you add another line of console

**Comment by mohammadsaju** [comment]
can you add another one

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('hello world');
```
Comment (mohammadsaju) [inline index.js:1]: can you change this with "i am going to be an software engineer"

Comment (rohan-sorkar) [reply index.js:1]: hmmm sure

**Review by mohammadsaju — APPROVED** [review]
well done @rohan-sorkar

**Pull request merged**